python/arcticdb/arctic.py

```diff
@@ -34,6 +35,10 @@ class Arctic:
         InMemoryLibraryAdapter,
     ]
 
+    # For test fixture clean up
+    _created_lib_names: Optional[List[str]] = None
```

Comment:
Why do we need this additional state? Can't we use the libraries (with prefixes) that we already have stored in the library manager, and strip the prefix on read, using the logic in `library_manager_names_to_user_facing` which we already need?

Reply:
In the `real_s3` fixture, there might be existing libraries created by other tests.

Reply:
It will be set on the instance level when it's set. This `None` is the default (when used in production code).

Reply:
But it should just be instance state, right?

Reply:
This is a pretty standard way to set defaults in Python? `self._created_lib_names` would get this `None` unless it's set in the instance.